### ISSUE TITLE
[wxwidgets] build failure

### ISSUE THREAD

**Issue by spiwocoal** [microsoft/vcpkg#10899]
**Host Environment**
- OS: Manjaro Linux 20.0 Lysia
- Compiler: revision

**To Reproduce**
`./vcpkg install wxwidgets`

**Failure logs** 
```
Computing installation plan...
The following packages will be built and installed:
    wxwidgets[core]:x64-linux
Starting package 1/1: wxwidgets:x64-linux
Building package wxwidgets[core]:x64-linux...
-- Using cached /home/bruno/vcpkg/downloads/wxWidgets-wxWidgets-v3.1.3.tar.gz
-- Using source at /home/bruno/vcpkg/buildtrees/wxwidgets/src/v3.1.3-91ca46db88
-- Configuring x64-linux-dbg
-- Configuring x64-linux-rel
-- Building x64-linux-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:136 (message):
    Command failed: /home/bruno/vcpkg/downloads/tools/cmake-3.14.0-linux/cmake-3.14.0-Linux-x86_64/bin/cmake --build . --config Debug --target install -- -v
    Working Directory: /home/bruno/vcpkg/buildtrees/wxwidgets/x64-linux-dbg
    See logs for more information:
      /home/bruno/vcpkg/buildtrees/wxwidgets/install-x64-linux-dbg-out.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_build_cmake.cmake:91 (vcpkg_execute_build_process)
  scripts/cmake/vcpkg_install_cmake.cmake:24 (vcpkg_build_cmake)
  ports/wxwidgets/portfile.cmake:38 (vcpkg_install_cmake)
  scripts/ports.cmake:90 (include)


Error: Building package wxwidgets:x64-linux failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: wxwidgets:x64-linux
  Vcpkg version: 2020.02.04-unknownhash

Additionally, attach any relevant sections from the log files above.
```
[install-x64-linux-dbg-out.log](https://github.com/microsoft/vcpkg/files/4498053/install-x64-linux-dbg-out.log)

**Additional context**
I know the problem is because in Arch Linux (and by consequence, Manjaro) harfbuzz headers are in /usr/include/harfbuzz, but for some reason, g++ doesn't scan by default that directory unless stated with `-I /usr/include/harfbuzz`, or the library being included with `#include <harfbuzz/hb.h`, but I'm not sure how to add that first option to vcpkg, and if I can make this work without having to modify the files, it would be really helpful.

**Comment by LilyWangL** [comment]
Thanks for posting this issue. You can install `harfbuzz:x64-linux` to temporarily solve this problem. I will confirm this issue soon.

**Comment by spiwocoal** [comment]
I am getting the exact same error, I installed `harfbuzz:x64-linux` with `./vcpkg install harbuzz`, but nothing changed.

**Comment by LilyWangL** [comment]
Closed as duplicate of #7931.
